elfi/methods/bo/acquisition.py

```diff
@@ -172,13 +172,10 @@ def evaluate(self, x, t=None):
         t : int
             Current iteration (starting from 0).
         """
-        if not isinstance(t, int):
-            raise ValueError("Parameter 't' should be an integer.")
```

Comment:
This should never happen from core side, so I would just let the _beta function raise the exception if the parameter is not interpretable as a value